scss/settings/_settings.scss

```diff
@@ -73,6 +73,7 @@ $global-radius: 0;
 $global-text-direction: ltr;
 $global-flexbox: false;
 $print-transparent-backgrounds: true;
+$contrast-warnings: true;
```

Comment:
> I'm assuming by adding the variable in the _color.scss file, it will automatically be generated in the _settings.scss file?

Yup that is the case dear!